packages/pages/src/common/src/feature/stream.test.ts

```diff
@@ -96,6 +96,16 @@ describe("formatSiteStream", () => {
 });
 
 describe("readSiteStream", () => {
+  afterEach(() => {
+    if (fs.existsSync("config.yaml")) {
+      fs.rmSync("config.yaml");
+    }
+    if (fs.existsSync("sites-config/site-stream.json")) {
+      fs.rmSync("sites-config/site-stream.json");
```

Comment:
You can delete the folder and everything in it in one line with `fs.rmSync(dir, { recursive: true, force: true })`